lib/reporter.js

```diff
@@ -0,0 +1,45 @@
+var chalk = require('chalk');
+var _ = require('lodash');
+var defaultFormatter = require('./defaultFormatter');
+
+module.exports = function(opts) {
+  var options = opts || {};
+
+  var formatter = options.formatter || defaultFormatter();
+
+  return function(css, result) {
+    var messagesToLog = (!options.plugins || !options.plugins.length)
+      ? result.messages
+      : result.messages.filter(function(message) {
+          if (options.plugins.indexOf(message.plugin) !== -1) {
```

Comment:
I mean if you don't like the bitwise shift you can always do:

```js
return options.plugins.indexOf(message.plugin) !== -1;
```

Or:

```js
return options.plugins.indexOf(message.plugin) > -1;
```

Personally I like the bitwise operator but I don't see its usage often. Either way the point I was trying to make was that you could just clean it up by returning the result of `indexOf`, rather than wrapping in an if statement.